doc/content/lorawan/db-dbm-dbi-dbd/_index.md

```diff
@@ -0,0 +1,100 @@
+
+## dB (decibel)
+
+The decibel can be used to express the ratio of two physical quantities such as power, sound intensity, sound pressure, voltage, and current on a logarithmic scale. In LoRaWAN we use decibel to express the ratio between two power levels usually given in Watt (W) or milliwatt (mW).
+
+The power ratio can be expressed in decibels using the formula, 
+
+**dB = 10 log<sub>10</sub> (P<sub>out</sub>/P<sub>in</sub>)**
+
+where P<sub>out</sub> is the output power and P<sub>in</sub> is the input power.
+
+{{< note "When we are dealing with the power levels we use 10log units." />}}
+
+For example, if an amplifier turns a 1 W signal into a 1000 W signal, its power ratio can be expressed as,
+
+dB = 10 log<sub>10</sub> (P<sub>out</sub>/P<sub>in</sub>) = 10 log<sub>10</sub> (1000/1) = 30 dB
+
+Decibel doesn’t provide an absolute value. By looking at the decibel value you can’t say the input and output power of a device or cable etc, but you can say whether it offers a gain or a loss.
+
+A power ratio greater than 0 dB is treated as a gain. For example, if an amplifier turns a 2 W signal into a 10 W signal, the power ratio is,
+
+10 log<sub>10</sub> P<sub>out</sub>/P<sub>in</sub>  = 10 log<sub>10</sub> 10/2   = 10 log<sub>10</sub> 5  = 6.9 dB (gain)
+
+A power ratio less than 0 dB is treated as a loss (negative gain or attenuation). For example, if 10 W of power is fed into a cable but only 8 W is measured at the output, the power ratio is,
+
+10 log<sub>10</sub> P<sub>out</sub>/P<sub>in</sub>  = 10 log<sub>10</sub> 8/10 = 10 log<sub>10</sub>(0.8) = -0.9 dB (loss)
+
+The power ratio of 0 dB means there is no gain or a loss.
+
+## dBm (decibel-milliwatt)
+
+If you use the reference input power of 1mW the power ratio can be expressed in dBm (‘m’ stands for milliwatt).
+
+**dBm = 10 log<sub>10</sub>(P<sub>out</sub> / 1 mW)**
+
+By using the above formula, P<sub>out</sub> can be expressed as an absolute value using W or mW as follows.
+
+**P<sub>out</sub> = 10<sup>(dBm/ 10)</sup>**
+
+For example, if a LoRaWAN gateway is rated as 22 dBm how much power does it generate in Watts?
+
+P<sub>out</sub> = 10<sup>(dBm/ 10)</sup>  = 10<sup>(22 / 10)</sup>  = 10<sup>(2.2)</sup> = 158.48 mW = 0.158 W
+
+### Rule of 10s and 3s
+
+The rule of 10s and 3s can be used easily to convert a dBm value to its corresponding absolute power value without using the logarithmic scale.
+
+- 10 dB = x10 (makes output power 10 times as the input power, for example, input=10 W and output=100 W)
+- -10 dB = ÷10 (makes output power 1/10 times as the input power, for example, input=100 W and output=10 W)
+- 3 dB = x2 (doubles the power, for example, input=5 W and output=10 W)
+- -3 dB = ÷2 (halves the power, for example, input=10 W and output=5 W)
+
+Only use 10s and 3s (positive or negaive) to express the given dBm value, for example, 1 dBm can be written as, 10 -3 -3 -3.
+
+Then apply the rule:
+
+1 dB = 10dB -3dB -3dB -3dB = x10 ÷2 ÷2 ÷2 = 1.25 mW
```

Comment:
We're mixing dB and dBm here.

1 dBm = 10dBm -3dBm -3dBm -3dBm = 1mW x10 ÷2 ÷2 ÷2 = 1.25 mW

Reply:
> Or should we keep to only ratios in this one? ex:
> 1 dB = 10dB -3dB -3dB -3dB = x10 ÷2 ÷2 ÷2 = x1.25
> and then show how the m in dBm affects it. ex:
> 1dBm = 1mW x1.25 = 1.25mW

This is better